app/images/assets/icon-download-text.svg

```diff
@@ -0,0 +1 @@
+<svg width="13" height="15" viewBox="0 0 13 15" xmlns="http://www.w3.org/2000/svg" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns"><title>icon-download</title><desc>Created with Sketch.</desc><path d="M7.498 11.62l.004.004 5.123-5.123-.004-.004c.236-.265.38-.615.38-.998 0-.828-.672-1.5-1.5-1.5-.448 0-.849.196-1.124.507l-2.376 2.376v-6.883h-3v6.879l-2.492-2.492-.001.001c-.266-.241-.619-.388-1.007-.388-.828 0-1.5.672-1.5 1.5 0 .387.147.741.388 1.007l-.001.001 5.123 5.123.002-.002c.264.231.61.372.988.372.383 0 .732-.144.998-.38zm-7.498 1.38v2h13v-2h-13z" sketch:type="MSShapeGroup" fill="#76899A"></path></svg>
```

Comment:
Do you think we need this meta data sketch leave behind `xmlns="http://www.w3.org/2000/svg" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns"`? That and the title?